plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/index.tsx

```diff
@@ -35,6 +66,10 @@ const Edit = ( props: ProductCollectionEditComponentProps ) => {
 				openCollectionSelectionModal={ () =>
 					setIsSelectionModalOpen( true )
 				}
+				productCollectionUIStateInEditor={
+					productCollectionUIStateInEditor
+				}
```

Comment:
I like the suggestion. I will pass a boolean prop instead of making further recognition of the UI state in ProductCollectionContent. Thanks for the suggestion. 🙌🏻